src/main/java/xyz/nucleoid/extras/lobby/item/TaterGuidebookItem.java

```diff
@@ -0,0 +1,231 @@
+package xyz.nucleoid.extras.lobby.item;
+
+import java.util.Set;
+
+import com.google.common.collect.HashMultimap;
+import com.google.common.collect.SetMultimap;
+
+import eu.pb4.polymer.core.api.item.PolymerItem;
+import eu.pb4.sgui.api.elements.BookElementBuilder;
+import eu.pb4.sgui.api.gui.BookGui;
+import net.minecraft.SharedConstants;
+import net.minecraft.entity.player.PlayerEntity;
+import net.minecraft.item.Item;
+import net.minecraft.item.ItemStack;
+import net.minecraft.item.Items;
+import net.minecraft.nbt.NbtElement;
+import net.minecraft.nbt.NbtHelper;
+import net.minecraft.nbt.NbtList;
+import net.minecraft.registry.Registries;
+import net.minecraft.screen.ScreenTexts;
+import net.minecraft.server.network.ServerPlayerEntity;
+import net.minecraft.text.ClickEvent;
+import net.minecraft.text.HoverEvent;
+import net.minecraft.text.Style;
+import net.minecraft.text.Text;
+import net.minecraft.text.Texts;
+import net.minecraft.util.Formatting;
+import net.minecraft.util.Hand;
+import net.minecraft.util.Identifier;
+import net.minecraft.util.TypedActionResult;
+import net.minecraft.util.math.BlockPos;
+import net.minecraft.world.World;
+import net.minecraft.world.chunk.Chunk;
+import xyz.nucleoid.extras.lobby.block.TinyPotatoBlock;
+import xyz.nucleoid.extras.mixin.lobby.ThreadedAnvilChunkStorageAccessor;
+
+public class TaterGuidebookItem extends Item implements PolymerItem {
+    private static final Text MISSING_SYMBOL = Text.literal("❌").formatted(Formatting.RED);
+    private static final Text FOUND_SYMBOL = Text.literal("✔").formatted(Formatting.GREEN);
+    private static final Text TOO_MANY_SYMBOL = Text.literal("✔").setStyle(Style.EMPTY.withColor(0x055005));
+
+    private static final int RECORD_COOLDOWN = 2 * SharedConstants.TICKS_PER_SECOND;
+
+    private static final String TATER_POSITIONS_KEY = "tater_positions";
+
+    public TaterGuidebookItem(Settings settings) {
+        super(settings);
+    }
+
+    @Override
+    public TypedActionResult<ItemStack> use(World world, PlayerEntity user, Hand hand) {
+        var stack = user.getStackInHand(hand);
+
+        if (!world.isClient() && user.isCreativeLevelTwoOp()) {
```

Comment:
Hmm, should the guidebook only be usable by ops? (is it meant to be used by players? at least, viewing it)